notification_service/settings.py

```diff
@@ -234,3 +253,6 @@
     with open(local_settings_path) as fp:
         code = compile(fp.read(), local_settings_path, "exec")
     exec(code, globals(), locals())
+
+
+SESSION_SERIALIZER = "django.contrib.sessions.serializers.PickleSerializer"
```

Comment:
Is this really needed? Thinking about possible attack vectors related to using PickleSerializer:
- https://docs.djangoproject.com/en/5.0/topics/http/sessions/#session-serialization
- https://docs.python.org/3/library/pickle.html#module-pickle
> Warning The pickle module is not secure. Only unpickle data you trust.
It is possible to construct malicious pickle data which will execute arbitrary code during unpickling. Never unpickle data that could have come from an untrusted source, or that could have been tampered with.

Reply:
If using PickleSerializer is not strictly needed perhaps using the default JSONSerializer instead would be safer?

https://docs.djangoproject.com/en/5.0/topics/http/sessions/#session-serialization
> we highly recommend sticking with JSON serialization especially if you are using the cookie backend.

Reply:
Don't remember where that came from. A quick test suggested that this can be removed. I need to investigate further.

Reply:
I'll leave that, because from [django-helusers repo](https://github.com/City-of-Helsinki/django-helusers):

> Helusers stores the access token expiration time as a datetime which is not serializable to JSON, so Django needs to be configured to use the built-in PickleSerializer.